views/partials/select-server-table.hbs

```diff
@@ -16,7 +16,7 @@
         <div class="selectServer__serverURL">{{server.displayName}}</div>
```

Comment:
/nit Happy to proceed on this PR without it. 

------------

Since this is for both apps and servers, the naming could be confusing.

For better readability, we can have different options:

1. Rename this to `select-table`, try to name things inside as generic as possible. 
2. Or Duplicate this file, so we have `select-server-table.hbs` and `select-apps-table.hbs`. Sometimes duplication is not a bad things. Consider in the future the server listing page and apps listing page may have more differences. --> And also remove the mapping `const gheServers = chain(groupBy(allServers, "gitHubBaseUrl")).map((_, key) => ({ displayName: key })).value();` into `displayName`. Map it to `xxxUrl`.
3. And/Or Rename the bit to be `{{item.identifier}}`, as neither it is always a `server` nor the field inside is always a `displayName` (it can be url)

Reply:
I'll do these changes in a separate PR..

Reply:
https://github.com/atlassian/github-for-jira/pull/1377